### ISSUE TITLE
Unable to use coinbase as rate provider

### ISSUE THREAD

**Issue by sha-265** [michaelWuensch/BitBanana#97]
## Description

When selecting coinbase as rate provider, the currency list is still mempool's list

## Expected Behavior

load currencies list from Coinbase.

## Actual Behavior

still using mempool list

## Steps to Reproduce

1. choose Coinbase as rate provider
2. try to add a currency that is available only on coinbase
3. see if it missing

## Context

 I'm unable to choose different curencies from Coinbase, that are missing from mempool.

## Your Environment

- BitBanana version:  0.8.3
- Android version: 13
- Device: pixel 4a
- LND Version: core 24.08

**Comment by sha-265** [comment]
@michaelWuensch yep, disabling Tor fixed the problem. maybe you should spilt this setting for rate provider, so I can use Tor for the other things. or add a rate provider with more currencies that not blocking tor. thanks

**Comment by michaelWuensch** [comment]
@sha-265 I have to reimplement tor anyway before I can release the next version as the current library is no longer compatible with the now required API level.
I'll keep this in mind and maybe make a Tor settings page where you can turn on and off tor for specific things.

**Comment by sha-265** [comment]
Thank you
